app/src/main/java/com/duckduckgo/app/referral/AppInstallationReferrerStateListener.kt

```diff
@@ -45,151 +26,3 @@ interface AppInstallationReferrerStateListener {
     }
 
 }
-
-@Singleton
```

Comment:
we can keep the `AppInstallationReferrerStateListener ` in the `main` folder, and each one of the variants will use its own implementation of the interface